settings.gradle.kts

```diff
@@ -100,13 +100,14 @@ include(
 )
 
 plugins {
-   id("com.gradle.enterprise") version "3.17.2"
+   id("com.gradle.develocity") version "3.17.4"
 }
 
-gradleEnterprise {
+develocity {
    buildScan {
-      termsOfServiceUrl = "https://gradle.com/terms-of-service"
-      termsOfServiceAgree = "yes"
+      termsOfUseUrl = "https://gradle.com/help/legal-terms-of-use"
+      termsOfUseAgree = "yes"
+      publishing.onlyIf { false }
```

Comment:
💬 `com.gradle.develocity` publishes scans by default. To enable on demand publishing for CI builds only, we need to disable default publication.